scripts/link-ts-references.ts

```diff
@@ -137,7 +138,8 @@ function saveConfigs(configs: PackageInfo[]) {
   configs.forEach(({ modified, path, tsConfig }) => {
     if (modified) {
       const tsConfigFile = join(path, "tsconfig.json");
-      writeFileSync(tsConfigFile, JSON5.stringify(tsConfig, null, 2));
+      const body = JSON5.stringify(tsConfig, null, 2) + EOL;
+      writeFileSync(tsConfigFile, body);
```

Comment:
This is the only meaningful change in this file (plus the import on line 8).

The rest is just auto-formatting prettier.

Reply:
Yes it is!